manifests/q/QNAP/NetBakReplicator/4.5.12.1108/QNAP.NetBakReplicator.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: QNAP.NetBakReplicator
+PackageVersion: 4.5.12.1108
+PackageLocale: en-US
+Publisher: QNAP

```

Comment:
```suggestion
Publisher: QNAP Systems, Inc.
```

Reply:
I will also need to update the PackageIdentifier to match correct.  If so would that not cause a issue with the comma ?

Reply:
The PackageIdentifier only looks at the Folder Structure, the Publisher and PackageName can be something entirely different (as long as it matches what Add and Remove programs report)
I am sorry if I confused you but this needs to match so WinGet upgrade/ WinGet list work correctly...

Reply:
makes sense I will make all the changes on all of them so you dont need to flag each one.

Reply:
Thank you 😁